R/check-inputs-scoring-functions.R

```diff
@@ -123,22 +123,27 @@ assert_input_interval <- function(observed, lower, upper, interval_range) {
   diff <- upper - lower
   diff <- diff[!is.na(diff)]
   if (any(diff < 0)) {
-    stop(
-      "All values in `upper` need to be greater than or equal to ",
-      "the corresponding values in `lower`"
+    cli_abort(
```

Comment:
This lack of coverage looks real to me. I think we should promote fixing this to an issue vs dealing with here.

Reply:
Unsure 🤷 
I specifically added this test: 
https://github.com/epiforecasts/scoringutils/blob/72d9090309d3fea37b396e25c2637d3b2521cbf1/tests/testthat/check-inputs-scoring-functions.R#L13
<img width="795" alt="image" src="https://github.com/epiforecasts/scoringutils/assets/37978797/69158a23-9634-42f8-8568-e1cbcbbf812f">
But I added an issue (https://github.com/epiforecasts/scoringutils/issues/686)